NitroxClient/GameLogic/Spawning/Bases/GhostEntitySpawner.cs

```diff
@@ -156,8 +157,7 @@ private static void MoveTransformToGhostEntity(Transform transform, GhostEntity
         }
         else
         {
-            transform.position = ghostEntity.Transform.LocalPosition.ToUnity();
-            transform.rotation = ghostEntity.Transform.LocalRotation.ToUnity();
+            transform.SetPositionAndRotation(ghostEntity.Transform.LocalPosition.ToUnity(), ghostEntity.Transform.LocalRotation.ToUnity());
```

Comment:
Is this the right behavior? Global = Local?

Reply:
There's currently an issue with NitroxTransform deserialization on client-side because its Parent field isn't reassigned so the Position will be the same as LocalPosition (and same goes for other fields).
I added a TODO comment in there to notice it and I hope this issue will be fixed sooner or later